overlord/hookstate/ctlcmd/refresh.go

```diff
@@ -114,6 +119,7 @@ func (c *refreshCommand) Execute(args []string) error {
 		{c.PrintInhibitLock, "--show-lock"},
 		{c.Hold, "--hold"},
 		{c.Proceed, "--proceed"},
+		{c.Tracking, "--tracking"},
```

Comment:
This check is also untested, there's already a test you can add this to in `refresh_test.go`